src/scripts/compute_impacted_targets.sh

```diff
@@ -31,19 +36,14 @@ if [[ -n ${BAZEL_STARTUP_OPTIONS} ]]; then
 fi
 logIfVerbose "Bazel startup options" "${bazel_startup_options}"
 
-_bazel() {
-	# trunk-ignore(shellcheck)
-	bazel ${bazel_startup_options} "$@"
-}
-
-# trunk-ignore(shellcheck)
-alias _java=$(_bazel info java-home)/bin/java
+curl --retry 5 -Lo bazel-differ "https://github.com/ewhauser/bazel-differ/releases/latest/download/bazel-differ-linux-${ARCH}"
```

Comment:
I think we should be explicit somewhere that we need to run on linux, whether we:
1. Check and make the action explicitly fail if that's not the case through something like uname (preferable)
2. Put it in the readme

Reply:
I think that'll automatically occur in prerequisites.sh -- we invoke `uname`, which will fail if on non-Linux machines.